Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
+    
+    //MARK: Properties
+    fileprivate var session : Session? = nil
```

Comment:
This is a good question :) Here is the difference between mobile app and backend, one app needs to process different requests at the same time. 
Let's discuss this on next meeting!